Backoffice/NavigationPanel/Strategies/GeneralNodesPanelStrategy.php

```diff
@@ -47,12 +47,24 @@ public function __construct(NodeRepositoryInterface $nodeRepository, CurrentSite
     public function show()
     {
         $siteId = $this->currentSiteManager->getCurrentSiteId();
-        $nodes = $this->nodeRepository->findLastVersionByType($siteId, NodeInterface::TYPE_TRANSVERSE);
+        $language = $this->currentSiteManager->getCurrentLocale();
```

Comment:
I think it's about time to rename this strategy: it can't be GeneralNodeS as it contains only one node. And Genereral isn't meaningfull, I'd rather prefer something like TransverseNodeStrategy